src/effects/VST3/VST3Effect.cpp

```diff
@@ -179,122 +159,34 @@ wxString GetFactoryPresetsPath(const VST3::Hosting::ClassInfo& effectClassInfo)
 
 }
 
-void VST3Effect::Initialize()
-{
-   using namespace Steinberg;
-
-   const auto& pluginFactory = mModule->getFactory();
-
-   auto effectComponent = pluginFactory.createInstance<Vst::IComponent>(mEffectClassInfo.ID());
-   if(!effectComponent)
-      throw std::runtime_error("Cannot create VST3 effect component");
-   if(effectComponent->initialize(&AudacityVst3HostApplication::Get()) != kResultOk)
-      throw std::runtime_error("Cannot initialize VST3 effect component");
-
-   auto audioProcessor = FUnknownPtr<Vst::IAudioProcessor>(effectComponent);
-   if(!audioProcessor)
-      //It's stated that "This interface must always be supported by audio processing plug-ins."
-      throw std::runtime_error("VST3 plugin does not provide audio processor interface");
-   
-   if(audioProcessor->canProcessSampleSize(Vst::kSample32) != kResultTrue)
-      throw std::runtime_error("32-bit sample size not supported");
-
-   mEffectComponent = effectComponent;
-   mAudioProcessor = audioProcessor;
-}
-
 EffectFamilySymbol VST3Effect::GetFamilySymbol()
 {
    return XO("VST3");
 }
 
-VST3Effect::VST3Effect(const VST3Effect& other)
-   : VST3Wrapper(other.mModule, other.mEffectClassInfo)
```

Comment:
Copy construction was eliminated, so perhaps you may also declare the copy with explicit `= delete`